crates/dojo-core/src/test_utils.cairo

```diff
@@ -29,25 +38,76 @@ fn spawn_test_world(components: Array<felt252>, systems: Array<felt252>) -> IWor
     ).unwrap();
     let world = IWorldDispatcher { contract_address: world_address };
 
+    // register auth components and systems
+    let (auth_components, auth_systems) = mock_auth_components_systems();
+    let mut index = 0;
+    loop {
+        if index == auth_components.len() {
+            break ();
+        }
+        world.register_component(*auth_components.at(index));
+        index += 1;
+    };
+
+    let mut index = 0;
+    loop {
+        if index == auth_systems.len() {
+            break ();
+        }
+        world.register_system(*auth_systems.at(index));
+        index += 1;
+    };
+
+    // Grant Admin role to the spawner
+    let caller = get_caller_address();
+    let mut grant_role_calldata: Array<felt252> = ArrayTrait::new();
+
+    grant_role_calldata.append(caller.into()); // target_id
+    grant_role_calldata.append('Admin'); // role_id
+    world.execute('GrantAuthRole'.into(), grant_role_calldata.span());
+
```

Comment:
`spawn_test_world` now is a bundled transaction that does the following

1. Create an empty world
2. Install the default authentication components and systems
3. Grants the `Admin` role
4. Register user's components and systems
5. Initialize the world with the given authentication routes